src/cider/tests/nextgen/functional/benchmark/process_log.sh

```diff
@@ -0,0 +1,74 @@
+#! /bin/bash
+# Copyright (c) 2022 Intel Corporation.
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# $1: log file dir
+# cd $1
+
+files=`ls *INFO*log`
+# process each log file
+for file_name in $files
+do
+# file_name="NextgenBenchmarkTest.INFO.20221221-071202.log"
+
+# replace * with \*
+sed -i 's/*/\*/g' $file_name
```

Comment:
Any introduction about how to use this script?

Reply:
I will try to integrate this file into jenkins workflow later. this script will process log files which contains time breakdown and get a csv file for each log file.